### ISSUE TITLE
Pipeline template loading never ends (4.3)

### ISSUE THREAD

**Issue by ppitonak** [openshift/console#4468]
* login **as non-admin** to OpenShift 4.3 with Pipelines 0.10.5 installed
* switch to developer perspective
* click *Add* and then *From Git*
* select one of builder images

expected:
* I see how my pipeline will look like

actual:
* spinner never disappears

![odc_pipeline_template](https://user-images.githubusercontent.com/445100/75252136-56dd8080-57dc-11ea-9b72-8c9519228be9.gif)

**Comment by ppitonak** [comment]
Seeem to be the same problem as in https://github.com/openshift/console/pull/3607

**Comment by vikram-raj** [comment]
@ppitonak Yes, I am going to cherry pick #3607 into release-4.3.

**Comment by andrewballantyne** [comment]
/assign

@ppitonak We've discussed this on a few occasions. Where is the testing with issue? I believe the Operator is the one with a fix here.

If needed, we can port our fix back to 4.3 that just hides the section when you lack permissions for it. @siamaksade and I discussed this and we agreed it was a hack that didn't really resolve the issue for the user. 

Let me know when you get a chance Pavol.
